### ISSUE TITLE
Necessito de implementação de código por todos

### ISSUE THREAD

**Issue by fkenjikamei** [Casterrr/dinorun#4]
Caros, neste projeto preciso ver que todos tenham significativa contribuição nos códigos do projeto. Ou seja, preciso ver os commits individuais, para entender em que parte cada um contribuiu.

**Comment by Casterrr** [comment]
Certo. Os commits são feitos por cada um de nós, mas cada pull request passa pela avaliação de algum outro integrante do grupo. Até o momento, somente eu e o Maelton realizamos commits alterando o código

**Comment by fkenjikamei** [comment]
Preciso ver que todos estão trabalhando em cima dos códigos.
